lib/iris/tests/unit/fileformats/grib/save_rules/test_shape_of_the_earth.py

```diff
@@ -28,12 +28,12 @@
 
 import numpy as np
 
-from iris.coord_systems import GeogCS, TransverseMercator, OSGB
+from iris.coord_systems import GeogCS, TransverseMercator, OSGB, RotatedGeogCS
 from iris.exceptions import TranslationError
 from iris.tests.unit.fileformats.grib.save_rules import GdtTestMixin
 
 from iris.fileformats.grib._save_rules import shape_of_the_earth
-
+from iris.fileformats.grib._save_rules import grid_definition_template_5
```

Comment:
This function should not be imported in this test module as a test module for this function already exists.